README.md

```diff
@@ -6,120 +6,285 @@
 [![PyPI version][pypi-version-shield]][pypi-version-url]
 
 
-<!-- TABLE OF CONTENTS -->
-## Table of Contents
+<!-- ABOUT THE PROJECT -->
 
-* [About the Project](#about-the-project)
-  * [Prerequisites](#prerequisites)
-  * [Installation](#installation)
-* [Usage](#usage)
-* [Roadmap](#roadmap)
-* [Contributing](#contributing)
-* [License](#license)
-* [Contact](#contact)
-* [Acknowledgements](#acknowledgements)
+PyTorch library with measures and metrics for various image-to-image tasks like denoising, super-resolution, 
+image generation etc. This easy to use yet flexible and extensive library is developed with focus on reliability 
+and reproducibility of results. Use your favourite measures as losses for training neural networks with ready-to-use 
+PyTorch modules.  
 
 
+<!-- GETTING STARTED -->
+### Getting started  
 
-<!-- ABOUT THE PROJECT -->
-## About The Project
+```python
+import torch
+from photosynthesis_metrics import ssim
 
-The project is intended to become a easy to use yet extensive library with metrics for 
-various image-to-image tasks like denoising, super-resolution, image generation etc.
+prediction = torch.rand(3, 3, 256, 256)
+target = torch.rand(3, 3, 256, 256)
+ssim_index = ssim(prediction, target, data_range=1.)
+```
 
 
-### Prerequisites
+<!-- MINIMAL EXAMPLES -->
+### Minimal examples
 
-* [PyTorch 1.3+](https://pytorch.org) :white_check_mark:  
+<!-- SSIM EXAMPLES -->
+<details>
+<summary>Structural Similarity (SSIM)</summary>
+<p>
 
-### Installation
+To compute SSIM index as measure, use lower case function from the library:
```

Comment:
Change to "as a measure" here and further along the text for SSIM, MS-SSIM and TV .